src/main/scala/org/broadinstitute/dsde/workbench/avram/db/AvramComponent.scala

```diff
@@ -0,0 +1,15 @@
+package org.broadinstitute.dsde.workbench.avram.db
+
+import java.sql.Timestamp
+import java.time.Instant
+
+import slick.jdbc.JdbcProfile
+
+import scala.concurrent.ExecutionContext
+
+trait AvramComponent {
+  val profile: JdbcProfile
+  implicit val executionContext: ExecutionContext
+
+  protected final val dummyDate: Instant = Instant.ofEpochMilli(1000)
```

Comment:
I think this came from Leo to work around a MySQL bug? Can probably remove this if it's not needed here.

Reply:
Actually, I think we'll need the same bug fix if we're using created/updated/destroyed dates for entities and collections :)